README.md

```diff
@@ -1,2 +1,61 @@
-# js-sdk
-Tozny Platform JavaScript Software Development Kit
+# E3DB JavaScript SDK Client Interface
+
+The Tozny End-to-End Encrypted Database (E3DB) is a storage platform with powerful sharing and consent management features.
+
+[Read more on our blog.](https://tozny.com/blog/announcing-project-e3db-the-end-to-end-encrypted-database/)
+
+This repository contains interfaces and code providing the core of the Tozny platform JS SDKs. It defines types, API
+behaviors, and more to keep interactions consistent across the various code bases, yet allowing each SDK to define the cryptography layer and any context specific helpers for use in the target environment.
+
+## Use
+
+Import this package
+
+```sh
+npm install --save e3db-client-interface
```

Comment:
@lkwdwrd Is now not a good time to switch things over from the e3db to the Tozny namespace?

Reply:
yes, this whole readme has not been updated and this example code is incorrect. For the purposes of this PR, this is out of scope. I'm planning on updating this and much more during the upcoming hack days.